tests/suite/test_annotations.py

```diff
@@ -156,15 +156,19 @@ def test_nginx_config_defaults(self, kube_apis, annotations_setup, ingress_contr
 
         assert "Strict-Transport-Security" not in result_conf
 
+        assert " 256k;" in result_conf
```

Comment:
this line looks unclear. I suggest either adding a comment that the line asserts the zone size or assert "zone name 256k;". the name of the zone is the same as the name of the upstream.

this comment applies to the other similar asserts.

Reply:
I'll add an assert "zone name 256k;". As for the assertions in the test_methods with `@pytest.mark.parametrize` annotation there is no way improve those without significant changes.